### DISCUSSION TITLE
Slack Workspace

### DISCUSSION THREAD

**Discussion by cklenk** [dnnsoftware/Dnn.Platform#3714] [General] [answered]
Is there a slack workspace already? If not, would it be worthwhile to have one? I would think it would be nice to have some members of the community in there. I personally think it would be helpful to be a part of. Let me know.

Thanks
Cameron

**Comment by cklenk** [comment]
Or a Microsoft team as another option.

**Comment by sleupold** [comment, marked as answer]
The DNN community teams are already using slack workspaces to coordinate their activities.
See https://dnncommunity.org/Community/Leadership-Team for team description.

**Comment by valadas** [comment]
The most open of those workspaces is Dnn Open Help where everyone can discuss, it is at https://join.slack.com/t/openhelp/shared_invite/zt-5oi1bydv-w0IAChDA3BxIVRsRs3fCDA

Then we have various more private ones for the leadership groups where it is by invite only.
